pom.xml

```diff
@@ -1534,11 +1538,11 @@
                 </goals>
                 <configuration>
                   <target unless="skipITs">
-                    <mkdir dir="${download.dir}" />
-                    <get src="http://sourceforge.net/projects/zanata/files/wildfly/${mojarra.module.name}/download" dest="${download.dir}/${mojarra.module.name}" skipexisting="true" />
-                    <get src="http://sourceforge.net/projects/zanata/files/wildfly/${hibernate.module.name}/download" dest="${download.dir}/${hibernate.module.name}" skipexisting="true" />
-                    <unzip src="${download.dir}/${mojarra.module.name}" dest="${appserver.home}" />
-                    <unzip src="${download.dir}/${hibernate.module.name}" dest="${appserver.home}" />
+                    <mkdir dir="${download.dir}"/>
+                    <get src="http://sourceforge.net/projects/zanata/files/wildfly/${mojarra.module.name}/download" dest="${download.dir}/${mojarra.module.name}" skipexisting="true"/>
+                    <get src="http://sourceforge.net/projects/zanata/files/wildfly/${hibernate.module.name}/download" dest="${download.dir}/${hibernate.module.name}" skipexisting="true"/>
+                    <unzip src="${download.dir}/${mojarra.module.name}" dest="${appserver.home}"/>
+                    <unzip src="${download.dir}/${hibernate.module.name}" dest="${appserver.home}"/>
```

Comment:
I don't think checkstyle checks. Note that some of the XML attributes have been reordered too.

The main thing is to watch your git diffs and avoid unnecessary churn, because it muddies the history.  You might want to check your IDE settings too.